src/events/message/index.ts

```diff
@@ -29,13 +29,20 @@ export default class MessageEvent extends Event {
 				message.content.trim().toLowerCase(),
 			)
 		) {
-			const pingMessage = await message.channel.send("Ping ?").catch(noop);
+			const pingMessage = await message.channel.send("Ping ? (Cela peut prendre jusqu'à ").catch(noop);
```

Comment:
```suggestion
			const pingMessage = await message.channel.send(`Ping ? *Cela peut durer ${REQEUST_TIMEOUT}*`).catch(noop);
```
(`REQUEST_TIMEOUT` is a constant declared in the upper scope : `parseInt(process.env.REQUEST_TIMEOUT, 10)` or `+process.env.TIMEOUT` :trollface:)